code/workspace/troubleshooting-provisioning-errors/example_sqs_queues.yaml

```diff
@@ -0,0 +1,27 @@
+AWSTemplateFormatVersion: "2010-09-09"
+
+Description: AWS CloudFormation workshop - troubleshooting provisioning errors - example SQS queues (uksb-1q9p31idr)
+
+Parameters:
+  QueueNamePrefix:
+    Type: String
+    Default: my-example-queue
+
+Resources:
+  ExampleDeadLetterQueue:
+    Type: AWS::SQS::Queue
+    Properties:
+      FifoQueue: true
+      QueueName: !Sub '${QueueNamePrefix}-dead-letter.fifo'
+
+  ExampleSourceQueue:
+    Type: AWS::SQS::Queue
+    Properties:
+      # This example source queue is configured as a standard queue,
+      # whilst the dead-letter queue resource uses `true` for the
+      # `FifoQueue` property instead; this will result in an error.
+      FifoQueue: false
+      RedrivePolicy:
+        deadLetterTargetArn: !GetAtt ExampleDeadLetterQueue.Arn
+        maxReceiveCount: 5
+      QueueName: !Sub '${QueueNamePrefix}-source.fifo'
```

Comment:
Please rename the file to `sqs-queue.yaml`

Reply:
Thank you, @rezabekf - will use `sqs-queues.yaml` as mentioned earlier.